src/cmakeTools.ts

```diff
@@ -416,13 +404,13 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
     /**
      * Whether CTest is enabled
      */
-    get ctestEnabled() {
-        return this._ctestEnabled.value;
+    get cTestEnabled() {
```

Comment:
I would rather it be `ctest` instead of `cTest`.  We do `cmake` instead of `cMake` everywhere.